src/Components/Form/FormFields/Autocomplete.tsx

```diff
@@ -236,24 +242,42 @@ export const Autocomplete = <T, V>(props: AutocompleteProps<T, V>) => {
                   >
                     {({ focus }) => (
                       <div className="flex flex-col">
-                        <div className="flex justify-between">
-                          <span>{option.label}</span>
-                          <span>{option.icon}</span>
-                        </div>
-                        {option.description && (
-                          <div
-                            className={classNames(
-                              "text-sm font-normal",
-                              option.disabled
-                                ? "text-secondary-700"
-                                : focus
-                                  ? "text-primary-200"
-                                  : "text-secondary-700",
+                        <div className="flex items-center">
+                          <div className="flex flex-col">
+                            <div className="relative">
+                              <Avatar
+                                className="mr-2 h-11 w-11 rounded-full"
+                                name={option.label}
+                                imageUrl={option.image}
+                              />
+                              <span className="absolute bottom-0 right-0 z-10">
+                                {option.icon}
+                              </span>
+                            </div>
+                          </div>
+                          <div className="w-full">
+                            <div className="flex w-full justify-between">
+                              <span>
+                                {"@"}
```

Comment:
@noufalrahim like said previously, this is a reusable component. Modifications made here will affect all other places. `@` prefix is only supposed to be present for `UserAutocompleteFormField` not any other usages.

Reply:
So where am I supposed to add "@"

Reply:
can I add it in utils formatName?
![Screenshot from 2024-10-23 16-57-05](https://github.com/user-attachments/assets/6b101e07-8fd5-46d2-bc64-a0028e834f02)

Reply:
The `username` should have the prefix `@`, not the name

Reply:
You can add it in `optionDescription` of the UserAutocompleteFormField right?

![Uploading image.png…]()